csharp-advanced-topics/StaticClasses/StaticClasses/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using StaticClasses;
+
+public class Program 
+{
+    static void Main(string[] args)
+    {
+        //static class implementation
+        //no instantiation needed
+        Student.Id = 1;
+        Student.Name = "John Doe";
+        Student.DateOfBirth = new DateTime(1994, 12, 31);
+        Console.WriteLine("The student using a static class: " + Student.StudentDetails());
+
+        //non-static class student implementation 
+        //the class has to be instantiated
+        CollegeStudent student = new CollegeStudent();
```

Comment:
You can improve this by using var, or new(). We prefer using var, but it's up to you.